Instastories.py

```diff
@@ -166,35 +168,35 @@ def download_today_stories(arr_ids, cookie, folder_path, mode_flag):
                     if not os.path.isfile(fn_video):
                         videos = element['video_versions']
                         video_url = videos[0]['url']
-                        print("Video URL: {}".format(video_url))
+                        logger.debug("Video URL: {}".format(video_url))
                         base64_media.append(get_media(video_url, fn_video, "video/mp4", username))
                         count_v += 1
                     else:
-                        print("Video media already saved")
+                        logger.debug("Video media already saved")
 
                 if element['media_type'] == 1:
                     fn_img = os.path.join(time_directory, str(media_id) + ".jpg")
                     if not os.path.isfile(fn_img):
                         pics = element['image_versions2']['candidates']
                         pic_url = pics[0]['url']
-                        print("Photo URL: {}".format(pic_url))
+                        logger.debug("Photo URL: {}".format(pic_url))
                         base64_media.append(get_media(pic_url, fn_img, "img/png", username))
                         count_i += 1
                     else:
-                        print("Video media already saved")
+                        logger.debug("Video media already saved")
 
             if mode_flag in ["all", "metadata"]:
                 if media_id not in json_stories_seen:  # Now save the metadata in a json file
                     json_stories_seen.add(media_id)
                     json_stories_saved.append(element)
                     new_metadata = True
-
+        logger.info(f"{len(items)} elements in {username} stories")
```

Comment:
This does not account for how many of those we actually got (eg duplicate).
Maybe store count_i count_v before and check the increment?

Logging something like:

DL: 10 Pic / 5 Videos out of 20media for USERNAME might be nicer!